### PR TITLE
ruby: add French translation, update page

### PR DESCRIPTION
**Pull request by noraj**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.fr/common/ruby.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Vérifie les erreurs de syntaxe d'un script Ruby donné :
+
+`ruby -c {{script.rb}}`
+
+- Démarre le serveur HTTP intégré sur le port 8080 dans le répertoire actuel :
+
+`ruby -run -e httpd`
+
+- Exécute localement un binaire ruby sans installer la bibliothèque requise dont il dépend :
```
Comment (kbdharun) [inline pages.fr/common/ruby.md:23]:
```suggestion
- Exécute localement un binaire Ruby sans installer la bibliothèque requise dont il dépend :
```

File: pages/common/ruby.md [hunk 2]
```diff
@@ -20,6 +20,10 @@
 
 `ruby -run -e httpd`
 
+- Locally execute a ruby binary without installing the required library it depends on:
```
Comment (kbdharun) [inline pages/common/ruby.md:23]:
```suggestion
- Locally execute a Ruby binary without installing the required library it depends on:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by blueskyson — APPROVED** [review]

File: pages/common/ruby.md [hunk 3]
```diff
@@ -20,6 +20,10 @@
 
 `ruby -run -e httpd`
 
+- Locally execute a Ruby binary without installing the required library it depends on:
+
+`ruby -I{{path/to/library_folder}} -r{{library_require_name}} {{path/to/bin_folder/bin_name}} {{arguments}}`
```
Comment (kbdharun) [inline pages/common/ruby.md:25]:
```suggestion
`ruby -I {{path/to/library_folder}} -r {{library_require_name}} {{path/to/bin_folder/bin_name}}`
```

File: pages.fr/common/ruby.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`ruby -c {{script.rb}}`
+
+- Démarre le serveur HTTP intégré sur le port 8080 dans le répertoire actuel :
+
+`ruby -run -e httpd`
+
+- Exécute localement un binaire Ruby sans installer la bibliothèque requise dont il dépend :
+
+`ruby -I{{chemin/vers/dossier_bibliothèque}} -r{{nom_chargement_bibliothèque}} {{chemin/vers/dossier_bin/nom_bin}} {{arguments}}`
```
Comment (kbdharun) [inline pages.fr/common/ruby.md:25]:
```suggestion
`ruby -I {{chemin/vers/dossier_bibliothèque}} -r {{nom_chargement_bibliothèque}} {{chemin/vers/dossier_bin/nom_bin}}`
```

**Review by kbdharun — COMMENTED** [review]
Sorry, for the late suggestion, I cross-checked the docs now and there is a space between the flags. Also, we can skip the arguments flag and keep this example concise.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**